### ISSUE TITLE
Indexer does not select the correct mp4 file for streaming when original upload is an mp4

### ISSUE THREAD

**Issue by GeoffFroh** [denshoproject/ddr-cmdln#199]
The videojs component used to play mp4 video streamed from IA is not optimized to render very high-rate video, resulting in unacceptable buffering. For example:

https://ddrstage.densho.org/ddr-densho-1020-13/

In this case, the clip will play without buffering when accessed directly from IA:

https://archive.org/details/ddr-densho-1020-13

The problem is with the logic in the indexer that pulls the files metadata from IA to create the streaming url in the ddr ES document. When the original mezzanine upload to IA is a high-bitrate MP4, the ingest workflow will automatically create a lower-bitrate streamable version. This version (which IA uses for its own streaming player) is named with the original basename with `.ia.mp4` appended.

For example, in the case of the entity above the original upload was named: 

`ddr-densho-1020-13-mezzanine-5c4e884556.mp4`

and the streamable version (found in the [IA files manifest](https://ia801507.us.archive.org/0/items/ddr-densho-1020-13/ddr-densho-1020-13_files.xml)) is named:

`ddr-densho-1020-13-mezzanine-5c4e884556.ia.mp4`

The logic that processes the files metadata from IA needs to properly identify the streamable MP4 when the original file is an MP4. 

(This issue is related to https://github.com/denshoproject/ddr-public/issues/166)

**Comment by GeoffFroh** [comment]
For reference, a files manifest from IA where the original is our base-case MPEG2: https://ia801209.us.archive.org/3/items/ddr-densho-1014-4-5/ddr-densho-1014-4-5_files.xml

**Comment by gjost** [comment]
I think I've got a fix for this as of `ddr-cmdln` commit `1413e8b`. I can pull it to a VM in the HQ so you guys can test it.  This should also address [ddrpublic/issues/166](https://github.com/denshoproject/ddr-public/issues/166).

**Comment by GeoffFroh** [comment]
@sarabeckman reports a possible regression issue related to: https://github.com/denshoproject/ddr-public/issues/163

**Comment by sarabeckman** [comment]
tested on [ddr-densho-1020-13](https://ddrstage.densho.org/ddr-densho-1020-13/), [ddr-csujad-30](https://ddrstage.densho.org/ddr-csujad-30/), [ddr-densho-1000-1](https://ddrstage.densho.org/interviews/ddr-densho-1000-1-1/), [ddr-densho-400](https://ddrstage.densho.org/ddr-densho-400/), [ddr-chi-1 ](https://ddrstage.densho.org/ddr-chi-1/)-- indexed to ddrstage with b2 flag.  This included both video & audio objects that are both VH & AV.  All these collections worked as expected. Also indexed [ddr-densho-10](https://ddrstage.densho.org/ddr-densho-10/) a solely photo/doc collection also worked as expected. 

[ddr-densho-122](https://ddrstage.densho.org/interviews/ddr-densho-122-4-1/) - the video player is wrong and I saw the regression issue mentioned above by @GeoffFroh. The interviews in `ddr-densho-122` are entities 4 through 29.

**Comment by GeoffFroh** [comment]
This issue occurs with video entities that are stream-only from IA

**Comment by GeoffFroh** [comment]
Stream-only content such as `ddr-densho-122-4-1` are displaying the embedded IA player rather than our own embedded player (the correct behavior). This is related to: https://github.com/denshoproject/ddr-cmdln/issues/186

**Comment by gjost** [comment]
Here are the objects I'm currently testing in [the unit tests](https://github.com/denshoproject/ddr-cmdln/blob/master/ddr/tests/test_archivedotorg.py):
```
    oid = 'ddr-densho-400-1'  # (mp3, simple entity)
    oid = 'ddr-csujad-30-19-1'  # (mp3, VH interview segment)
    oid = 'ddr-densho-1000-1-1'  # (mp4 standard VH that we prepared)
    oid = 'ddr-densho-1020-13'  # (IA streaming video)
    oid = 'ddr-densho-122-4-1'  # (external video, stream-only)
```

**Comment by gjost** [comment]
Fixed as of commit `4ffb959` on `ddr-cmdln` `develop` branch.
